packages/forma-36-react-components/src/components/Accordion/AccordionPanel/AccordionPanel.tsx

```diff
@@ -26,29 +25,54 @@ export const AccordionPanel: FC<AccordionPanelProps> = ({
 }: AccordionPanelProps) => {
   const panelEl = useRef<HTMLDivElement>(null);
 
+  const getPanelContentHeight = () => {
+    const { current } = panelEl;
+
+    if (!current) {
+      // to keep the function return type as string only and
+      // not overcomplicate things with non-nullable checks
+      return '0px';
+    }
+
+    return `${current.scrollHeight / 16}rem`; // converting height pixels into rem
```

Comment:
This should probably use the `fontBaseDefault` token.

Reply:
Fixed, thanks for the comment.